sale_layout_category_hide_detail/views/invoice_report_templates.xml

```diff
@@ -33,7 +33,7 @@
             />
     </xpath>
     <xpath
-            expr="//tbody[hasclass('invoice_tbody')]//tr[hasclass('is-subtotal', 'text-right')]"
+            expr="//tbody[hasclass('invoice_tbody')]//tr[hasclass('is-subtotal', 'text-end')]"
```

Comment:
Why `right` is not working?

Reply:
that class was changed in the base template sale.report_saleorder_document:
```
 <tr class="is-subtotal text-end">

Reply:
Updated missing translation

Reply:
This is still not working:

![Selección_057](https://github.com/OCA/sale-reporting/assets/7165771/bb1b918e-3b92-4fc1-a7d7-755ca0364978)